README.md

```diff
@@ -161,6 +161,16 @@ If you want to send one or more files along, you can pass in a File object as da
 client.execute :foo, :bar, user: :johndoe, data: {file: File.new('/tmp/test.foo')}, format: :urlencoded
 ```
 
+**Package desciption**
+
+To access the entire content of a package's DESCRIPTION file
+```Ruby
+client.description 'mypackage'
```

Comment:
Voor execute geven de package in de voorbeelden als symbol door. Dus mooier om hier ook te doen (beide moet supported zijn).